sql/ddl_api.sql

```diff
@@ -214,8 +214,8 @@ CREATE OR REPLACE FUNCTION  set_replication_factor(
 AS '@MODULE_PATHNAME@', 'ts_hypertable_distributed_set_replication_factor' LANGUAGE C VOLATILE;
 
 -- Refresh a continuous aggregate across the given window.
-CREATE OR REPLACE FUNCTION refresh_continuous_aggregate(
+CREATE OR REPLACE PROCEDURE refresh_continuous_aggregate(
```

Comment:
i think this requires some discussion which parts we want as procedure and which parts we want as function, while a procedure allows transaction control it has some disadvantages and is gives less flexibility to the caller.

Reply:
This was discussed in design docs and if we are to retain the old functionality of how the invalidation threshold is set there is no real option.